api/src/main/java/org/pmiops/workbench/compliance/ComplianceServiceImpl.java

```diff
@@ -1,14 +1,20 @@
 package org.pmiops.workbench.compliance;
 
+import java.util.HashMap;
```

Comment:
Hmm. Somehow, the workspace mapper impl file lives in a `genenrated` dir, which explians why it doesn't get picked up: `workbench/api/build/generated/sources/annotationProcessor/java/main/org/pmiops/workbench/utils/WorkspaceMapperImpl.java`

I don't believe we want to . mask `*Impl.java`, as it would match lots fo our implementation services.

Reply:
What is this comment in reference to? I don't think it is actually about importing `HashMap`